pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub,  makeDesktopItem, ncurses, libX11, boost, cmake }:
+
+let
+  pname = "tome2";
+  description = "A dungeon crawler similar to Angband, based on the works of Tolkien.";
+
+  desktop = makeDesktopItem {
+    desktopName = pname;
+    name = pname;
+    exec = "@out@/bin/${pname}-x11";
+    icon = "${pname}";
+    terminal = "False";
+    comment = description;
+    type = "Application";
+    categories = "Game;RolePlaying;";
+    genericName = pname;
+  };
+
+in stdenv.mkDerivation rec {
```

Comment:
It doesn't appear that `rec` is actually needed here, though I could be missing something.